cmake/BLTMacros.cmake

```diff
@@ -480,18 +480,8 @@ macro(blt_patch_target)
         endif()
     endif()
 
-    # PGI does not support -isystem
-    if( (${arg_TREAT_INCLUDES_AS_SYSTEM}) AND (NOT "${CMAKE_CXX_COMPILER_ID}" STREQUAL "PGI"))
-        get_target_property(_target_includes ${arg_NAME} INTERFACE_INCLUDE_DIRECTORIES)
-        # Don't copy if the target had no include directories
-        if(_target_includes)
-            if(_standard_lib_interface)
-                target_include_directories(${arg_NAME} SYSTEM ${_scope} ${_target_includes})
```

Comment:
The scope here doesn't make sense and so was removed.  `PUBLIC` is  `INTERFACE` (inheriting to downstream targets) and `PRIVATE` (used when building itself). The latter doesn't make sense for system includes.